scss/main.scss

```diff
@@ -295,9 +296,45 @@ body {
 }
 
 // *** intro section ***
-.about-image {
-  width: 5%;
-  height: 5%;
+.about-text {
+  padding-bottom: 2.5rem;
+}
+
+.polaroid-wrapper {
+  text-align: center;
+}
+
+.polaroid {
+  width: 15rem;
+  height: auto;
+  position: absolute;
+  background-color: $polaroid-border;
+  padding: 1rem 1rem 3.25rem 1rem;
+
+  &.polaroid1{
+    right: 4rem;
+    top: -7rem;
+    transform: rotate(-3deg)
+  }
+
+  &.polaroid2{
+    right: -4rem;
```

Comment:
can u move both images to the left a bit pls - the right side of polaroid #2 isn't aligning w the right side of the following sections:
![Screen Shot 2021-08-12 at 6 16 35 PM](https://user-images.githubusercontent.com/62481939/129277350-1a969721-e3b7-4692-9fd8-4d27565cc88d.png)